### DISCUSSION TITLE
Image Tint

### DISCUSSION THREAD

**Discussion by Nexerate** [dotnet/maui#7419] [Ideas] [answered]
It would be useful to have the ability to tint images in MAUI. Common practice is to make icons white, and then tint them later. This ensures reusability, and saves time when you create icons.

Something like this:
`<Image
    Source="image.png"
    ImageTint="Gray"/>`

**Comment by pictos** [comment, marked as answer]
@Nexerate, this feature is on XCT.Compat (the XamarinCoomunityToolkit version that targets .NET MAUI, using the compatibility layer), and will be ported to CommunityToolkit.MAUI soon-sh
